### DISCUSSION TITLE
Donation Method?

### DISCUSSION THREAD

**Discussion by ykhurshid** [Schneegans/Fly-Pie#93] [Ideas & Feature Discussions]
This application has the handy distinction of both being genuinely useful and having strong branding potential. Therefore, it might be a smart idea to have some option for sending donations. I don't have any strong preferences for methods but it would be nice to have an option other than Paypal just because of how much of a cut they now take. 

I'm kind of broke for the moment but I was honestly looking around the GitHub page for an option to at least send you enough for a coffee.

**Comment by Schneegans** [comment]
Thank you very much for the kind words! I will definitely consider this! If there are any news on this, I'll let you know :wink:

**Comment by Schneegans** [comment]
I managed to set up GitHub sponsorships for my account. There is now a "Sponsor" button on Fly-Pie's repository page which lets people support the project with as little as 1$/month. I will add the sponsors to the README and the about dialog of Fly-Pie to keep things transparent. Let's see if someone is interested :smile:!

Do you have another donation platform in mind - besides Paypal - which could be of interest?

**Reply by ykhurshid** [reply]
In the US, they have Stripe but for international donors such as Me, bank details would be ideal but that's obviously a privacy risk unless if you've set up a business account (which might be overkill this early in the project). 

I'm not completely against donating through PayPal, I just think it's not necessarily the most optimal choice. However, they do have a huge userbase so your own cost-benefit analysis may conclude that the ubiquity of Paypal and the increased likelihood of donations may outweigh its cons.

Cryptocurrency might be a wise option to have available for those that do use it (I don't).
